runtime/signals.c

```diff
@@ -129,9 +118,6 @@ CAMLexport void caml_record_signal(int signal_number)
   atomic_fetch_add_explicit
     (&caml_pending_signals[signal_number], 1, memory_order_seq_cst);
```

Comment:
As far as I can see, it's enough to atomically store 1 in `caml_pending_signals[signal_number]`.  Plus, this avoids the highly theoretical risk of overflow and wrapping around to 0...

Reply:
Yes, this makes sense. Changing.